.mailmap

```diff
@@ -0,0 +1,22 @@
+Alex Lindsay <adlinds3@ncsu.edu>
+Alex Lindsay <lindsayad@users.noreply.github.com>
+Alex Lindsay <alexlindsay239@gmail.com>
+Casey Icenhour <cticenho@ncsu.edu>
```

Comment:
@cticenhour Do you still have access to your NCSU email? I heard NCSU deactivated soon after someone graduates.